{"uid":"7fc7e958f06ea218","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1702862056740,"stop":1702862067004,"duration":10264},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5568a02b6d33 <unknown>\n#1 0x55689ff73f87 <unknown>\n#2 0x55689ffc3c0e <unknown>\n#3 0x55689ffc3d11 <unknown>\n#4 0x5568a000f004 <unknown>\n#5 0x55689ffed36d <unknown>\n#6 0x5568a000c297 <unknown>\n#7 0x55689ffed0e3 <unknown>\n#8 0x55689ffb5044 <unknown>\n#9 0x55689ffb644e <unknown>\n#10 0x5568a027b861 <unknown>\n#11 0x5568a027f785 <unknown>\n#12 0x5568a0269285 <unknown>\n#13 0x5568a028041f <unknown>\n#14 0x5568a024d20f <unknown>\n#15 0x5568a02a4028 <unknown>\n#16 0x5568a02a41f7 <unknown>\n#17 0x5568a02b5ed4 <unknown>\n#18 0x7f48a0294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f32ce915490>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"dbb0e9800699a5426840900df3f26d71\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f32ce9e1f80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5568a02b6d33 <unknown>\nE       #1 0x55689ff73f87 <unknown>\nE       #2 0x55689ffc3c0e <unknown>\nE       #3 0x55689ffc3d11 <unknown>\nE       #4 0x5568a000f004 <unknown>\nE       #5 0x55689ffed36d <unknown>\nE       #6 0x5568a000c297 <unknown>\nE       #7 0x55689ffed0e3 <unknown>\nE       #8 0x55689ffb5044 <unknown>\nE       #9 0x55689ffb644e <unknown>\nE       #10 0x5568a027b861 <unknown>\nE       #11 0x5568a027f785 <unknown>\nE       #12 0x5568a0269285 <unknown>\nE       #13 0x5568a028041f <unknown>\nE       #14 0x5568a024d20f <unknown>\nE       #15 0x5568a02a4028 <unknown>\nE       #16 0x5568a02a41f7 <unknown>\nE       #17 0x5568a02b5ed4 <unknown>\nE       #18 0x7f48a0294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1702862016943,"stop":1702862016943,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1702862016943,"stop":1702862016943,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1702862016943,"stop":1702862016943,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5568a02b6d33 <unknown>\n#1 0x55689ff73f87 <unknown>\n#2 0x55689ffc3c0e <unknown>\n#3 0x55689ffc3d11 <unknown>\n#4 0x5568a000f004 <unknown>\n#5 0x55689ffed36d <unknown>\n#6 0x5568a000c297 <unknown>\n#7 0x55689ffed0e3 <unknown>\n#8 0x55689ffb5044 <unknown>\n#9 0x55689ffb644e <unknown>\n#10 0x5568a027b861 <unknown>\n#11 0x5568a027f785 <unknown>\n#12 0x5568a0269285 <unknown>\n#13 0x5568a028041f <unknown>\n#14 0x5568a024d20f <unknown>\n#15 0x5568a02a4028 <unknown>\n#16 0x5568a02a41f7 <unknown>\n#17 0x5568a02b5ed4 <unknown>\n#18 0x7f48a0294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f32ce915490>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"dbb0e9800699a5426840900df3f26d71\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f32ce9e1f80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5568a02b6d33 <unknown>\nE       #1 0x55689ff73f87 <unknown>\nE       #2 0x55689ffc3c0e <unknown>\nE       #3 0x55689ffc3d11 <unknown>\nE       #4 0x5568a000f004 <unknown>\nE       #5 0x55689ffed36d <unknown>\nE       #6 0x5568a000c297 <unknown>\nE       #7 0x55689ffed0e3 <unknown>\nE       #8 0x55689ffb5044 <unknown>\nE       #9 0x55689ffb644e <unknown>\nE       #10 0x5568a027b861 <unknown>\nE       #11 0x5568a027f785 <unknown>\nE       #12 0x5568a0269285 <unknown>\nE       #13 0x5568a028041f <unknown>\nE       #14 0x5568a024d20f <unknown>\nE       #15 0x5568a02a4028 <unknown>\nE       #16 0x5568a02a41f7 <unknown>\nE       #17 0x5568a02b5ed4 <unknown>\nE       #18 0x7f48a0294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1702862056740,"stop":1702862067004,"duration":10264},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5568a02b6d33 <unknown>\n#1 0x55689ff73f87 <unknown>\n#2 0x55689ffc3c0e <unknown>\n#3 0x55689ffc3d11 <unknown>\n#4 0x5568a000f004 <unknown>\n#5 0x55689ffed36d <unknown>\n#6 0x5568a000c297 <unknown>\n#7 0x55689ffed0e3 <unknown>\n#8 0x55689ffb5044 <unknown>\n#9 0x55689ffb644e <unknown>\n#10 0x5568a027b861 <unknown>\n#11 0x5568a027f785 <unknown>\n#12 0x5568a0269285 <unknown>\n#13 0x5568a028041f <unknown>\n#14 0x5568a024d20f <unknown>\n#15 0x5568a02a4028 <unknown>\n#16 0x5568a02a41f7 <unknown>\n#17 0x5568a02b5ed4 <unknown>\n#18 0x7f48a0294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"ed195a383a61e6c2","name":"public ip address","source":"ed195a383a61e6c2.txt","type":"text/plain","size":13},{"uid":"65d16dd31a3e42b7","name":"Cookies","source":"65d16dd31a3e42b7.json","type":"application/json","size":3356},{"uid":"24e5e0bcc49f2d15","name":"Session Storage","source":"24e5e0bcc49f2d15.json","type":"application/json","size":358},{"uid":"5dd9f55da8d31823","name":"Local Storage","source":"5dd9f55da8d31823.json","type":"application/json","size":208},{"uid":"1cfb15186f9fa7d6","name":"Console Logs","source":"1cfb15186f9fa7d6.json","type":"application/json","size":2},{"uid":"bd1ff9893191f3ef","name":"Full Page Screenshot","source":"bd1ff9893191f3ef.png","type":"image/png","size":1275914},{"uid":"49ee85951177204b","name":"URL","source":"49ee85951177204b.uri","type":"text/uri-list","size":23},{"uid":"43d6000f4e165a76","name":"Network Logs","source":"43d6000f4e165a76.json","type":"application/json","size":22849}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1251-712"},{"name":"thread","value":"2287-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"7fc7e958f06ea218.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}